CONTRIBUTING.md

```diff
@@ -68,38 +69,97 @@ Please see `./ci/features`.
 To test the usage of the Conjur Service Broker within a CF deployment, you can
 follow the demo scripts in the [Cloud Foundry demo repo](https://github.com/conjurinc/cloudfoundry-conjur-demo).
 
-To run the test suite on your local machine:
+#### Running the Dev Environment
+To test your changes within a running instance of Cloud Foundry and Conjur, run:
+
+```shell script
+./ci/start_dev_environment
+```
+
+This starts Conjur and Cloud Foundry containers, and provides terminal 
+access to the Cloud Foundry container. You do not need to restart the container
+after you make changes to the project.
+
+To run the `cucumber` within the development environment, run the following command from
+the `ci` directory, within the container:
+
+```shell script
+cucumber \
+    --format pretty \
+    --format junit \
+    --out ./features/reports \
+    --tags 'not @integration'
 ```
-$ ./package.sh   # Create the conjur-env binary in the vendor dir and a ZIP of the project contents
-$ ./test.sh      # Run the test suite
+
+#### Setup
+To run the test suite on your local machine, first run:
+```shell script
+./package.sh
 ```
+This will create the conjur-env binary in the vendor dir and a ZIP of the project contents.
 
 #### Integration Testing
```

Comment:
According to the wording in Issue #94, the `test_e2e` script should run only the End-to-End tests that require access to an external PCF/Tanzu. That is, it shouldn't be running the tests that `test_integration` script is doing.  So I'm thinking it probably makes sense to split this `Integration Testing` section into 2 sections:
- `Local Integration Testing` (or `Light-Weight Integration Testing`???)
- `End-to-End Testing`

Or if there's a better name for `Local Integration Testing`? I figured since it's associated with the `test_integration` script, we should stay consistent.